tests/Unit/PushwooshMessageTest.php

```diff
@@ -271,4 +271,32 @@ public function testTimezoneStrategyModification()
         $message = (new PushwooshMessage)->useRecipientTimezone();
         $this->assertFalse($message->jsonSerialize()['ignore_user_timezone']);
     }
+
+    /**
+     * Test modification of the silent parameters .
+     *
+     * @return void
+     */
+    public function testSilentModification()
+    {
+        $message = new PushwooshMessage();
+        $message->silent(true);
+        $this->assertArrayHasKey('ios_silent', $message->jsonSerialize());
+        $this->assertEquals(1, $message->jsonSerialize()['ios_silent']);
+        $this->assertArrayHasKey('android_silent', $message->jsonSerialize());
+        $this->assertEquals(1, $message->jsonSerialize()['android_silent']);
+    }
+
+    /**
+     * Test modification of the silent parameters .
+     *
+     * @return void
+     */
+    public function testSilentModificationNull()
+    {
+        $message = new PushwooshMessage();
+        $message->silent();
```

Comment:
This way we test the regression of https://github.com/laravel-notification-channels/pushwoosh/pull/20#discussion_r1946642523
```suggestion
        $message->silent()->silent(false);
```

Reply:
fixed